ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -17,78 +19,132 @@ import {RecentWork} from 'app/pages/homepage/recent-work';
 import {getRegistrationTasksMap, RegistrationDashboard} from 'app/pages/homepage/registration-dashboard';
 import {profileApi} from 'app/services/swagger-fetch-clients';
 import colors from 'app/styles/colors';
+import {Column, Row} from 'app/styles/flex';
 import {hasRegisteredAccessFetch, reactStyles, ReactWrapperBase, withUserProfile} from 'app/utils';
+import {environment} from 'environments/environment';
 import {
   BillingProjectStatus,
   Profile,
 } from 'generated/fetch';
 
 export const styles = reactStyles({
   mainHeader: {
```

Comment:
Meta-comment (no action required): whenever we start to have massive lists of object properties like we see here, I always, always favor alphabetizing the properties. Unless there's a very compelling reason to do otherwise, I've found alphabetization is the only sane way to organize lists like this.

This is a pretty pervasive style nitpick though, and one which could / should easily be cleaned up at some point in the future in a massive sweep of the codebase. I just can't help myself from commenting when I come across examples which I'd love to clean up someday :)

Reply:
They actually are alphabetized, but the linter has them alphabetized by file path rather than by object name.

Reply:
Ah, my comment was about the list of object entries within this reactStyles({ ... }) call, not the file imports themselves.

Reply:
I can see this being a better option for long term maintainability. I sorted the lines that we will be keeping (but not the lines that will be deleted when the feature flag is removed).